packages/terra-form-field/src/Field.scss

```diff
@@ -0,0 +1,75 @@
+@import '~terra-mixins';
+@import './cerner-mock-theme';
+
+:local {
+  .field {
+    border: 0;
+    margin-bottom: var(--terra-form-field-margin-bottom, 1.428em);
+    max-width: 1020px;
+    padding: 0;
+  }
+
+  .label-group {
+    font-family: var(--terra-form-field-label-font-family, 'Helvetica Neue');
+    font-size: var(--terra-form-field-label-font-size, 14px);
```

Comment:
We should set this value in rems so it scales if the user adjusts the default font-size. Set to `1rem`

Reply:
Ah totally missed converting these.